guiEditor/src/guiNodeTools.ts

```diff
@@ -17,6 +17,8 @@ import { ScrollViewer } from "babylonjs-gui/2D/controls/scrollViewers/scrollView
 import { RadioButton } from "babylonjs-gui/2D/controls/radioButton";
 import { ImageBasedSlider } from "babylonjs-gui/2D/controls/sliders/imageBasedSlider";
 
+const imageUrl = "https://playground.babylonjs.com/textures/grass.png";
```

Comment:
Should be either exported as a let or a static property of GUINodeTools to be "updatable" by the user